docs/001_develop/03_client-capabilities/004_header/02_customise.mdx

```diff
@@ -0,0 +1,170 @@
+---
+title: 'Customise Header'
+sidebar_label: 'Customise'
+id: client-header-customise
+keywords: [header, customise]
+tags:
+- header
+- header customise
+sidebar_position: 2
+---
+
+You can customise:
+
+- the icon shown on the navigation bar and flyout menu (this shows the Genesis logo by default)
+- navigation links at the left-hand side of the navigation bar
+- the control buttons on the right-hand side of the navigation bar; these can be shown or hidden, and their behaviour controlled via event listeners
+- the contents of the flyout menu
+
+### Examples
+
+Here is an example of the navigation bar with three navigation items, and all three control buttons shown.
+![Header with the standard genesis logo](/img/001_develop/03_client-capabilities/004_header/foundation-header-standard.png)
+
+This next example is the same as the previous example, except the Genesis logo is replaced with a custom icon.
+![Header with a customised logo](/img/001_develop/03_client-capabilities/004_header/foundation-header-replaced-img.png)
+
+In this next example, we have put a set of example options set in the flyout menu.
+![The sidebar included with the header opened with some example content](/img/001_develop/03_client-capabilities/004_header/foundation-header-sidebar.png)
+
+## Header set-up
+
+A lot of the Genesis seed apps come with the Header set up by default. To verify, you can do a text search in the client code for the `<foundation-header>` tag.
+
+### Icon
+
+By default, the navigation bar and flyout menu show the Genesis logo. You can override this by setting the `logoSrc` attribute. For example:
+
+```html
+<foundation-header logoSrc="https://icotar.com/avatar/genesis"></foundation-header>
+```
+
+The `logoSrc` defines the image that you want to display. Adding this attribute updates the logo on both the flyout and navigation bar. If you want to keep the Genesis logo, just omit this attribute.
+
+### Navigation items
+
+You can add navigation items to the left-hand side of the navigation bar. For each element, you can set `slot="routes"` attribute, so that navigation is controlled via a `@click` event. The following is a really basic example for adding a 'Home' button:
```

Comment:
Can the example below have two buttons to demonstrate that you can add two items? Can you add two buttons with `slot="routes"`